### PR TITLE
Fix going home

### PR DESCRIPTION
**Pull request by Alastair-L**
### Issue #:
https://github.com/beyondessential/tupaia-backlog/issues/770
PR 10/(11?) here: https://github.com/beyondessential/tupaia/pull/1063
### Changes:
When you log out or in, or click the "home" button, you are reset to the project splash screen (or the login screen, depending on if you are logged in), except if you are 'autologgingin' on app start, in which case you go where 'reactToInitialState' directs you.

### CODE REVIEW

File: packages/web-frontend/src/App.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ class App extends Component {
 
     const { dispatch } = store;
     dispatch(fetchInitialData());
+    dispatch(findLoggedIn(true));
```
Comment (Alastair-L) [inline packages/web-frontend/src/App.js:51]: We are using the 'shouldCloseDialog' key here... Is this fine or should we create a new, separate one? I think it is fine as is but wanted to flag it.

File: packages/web-frontend/src/containers/UserMenu/index.js [hunk 2]
```diff
@@ -45,10 +45,6 @@ class UserMenu extends Component {
     };
   }
 
-  componentDidMount() {
-    this.props.onRefreshCurrentUser();
-  }
```
Comment (Alastair-L) [inline packages/web-frontend/src/containers/UserMenu/index.js:50]: I couldn't find a reason that the user should only log in if the UserMenu renders, can you?

File: packages/web-frontend/src/App.js (hunk 1 above)
Comment (kael89) [reply packages/web-frontend/src/App.js:51]: As discussed, we should rename this variable to `loginType` and decouple login logic from UX logic. Each client of `findLoggedIn` can decide on how to manage the UX based on the `loginType`

File: packages/web-frontend/src/containers/UserMenu/index.js (hunk 2 above)
Comment (kael89) [reply packages/web-frontend/src/containers/UserMenu/index.js:50]: I can't find either!

Comment (kael89) [reply packages/web-frontend/src/containers/UserMenu/index.js:50]: This is the commit that added the above (3 years ago!): https://github.com/beyondessential/tupaia-web/commit/6fd61f54077e600dafea12ed4f35954e6b933ce3#diff-5408a22f0a6eb54adba94ad67d94d505

Comment (Alastair-L) [reply packages/web-frontend/src/containers/UserMenu/index.js:50]: Wow. Good find! Simpler times...

**Review by kael89 — APPROVED** [review]
Me gusta!!

**Pull request merged**